### ISSUE TITLE
Benchbark result？

### ISSUE THREAD

**Issue by Liu-Feng-deeplearning** [Orfium/bytecover#22]
Thanks for the author for the bytecover first, it is a great work!!

And i want to know if there are some results on benchmark datasets, such as shs100k? Can we get the same accuracy with original papers?

Thank you ~

**Comment by petvav** [comment]
Hello @Liu-Feng-deeplearning ,

Thank you for your interest. We have evaluated the trained model on the shs100k test dataset. Unfortunately, we couldn't achieve the results that the paper claims. This could be due to the ambiguities of the paper, described in the [README](https://github.com/Orfium/bytecover/blob/master/README.md). You can find our results on the shs100k test dataset at: https://github.com/Orfium/bytecover/issues/6#issuecomment-1647797855

**Comment by Bloos** [comment]
I've found another difference between your implementation and the paper. The default config has a compress_ratio of 20 but the paper says they used 100. This is quite a difference

**Comment by petvav** [comment]
Hello @Bloos ,
That is correct. We tested the 100 ratio but the model performed worse and our intuition was that the number 100 is a very large number and it truncates a lot of information. Instead, we used the ratio of 20, studied also in this [work](https://arxiv.org/pdf/1911.00334) (Section 2.2).

**Comment by Bloos** [comment]
did you train from scratch or from a pretrained resnet50 checkpoint?

**Comment by petvav** [comment]
From scratch
